File: SmartWash/src/main/java/com/smartwash/vo/user_coupon/UserCouponVo.java
package com.smartwash.vo.user_coupon;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.Data;

import java.time.LocalDateTime;

@Data
public class UserCouponVo {
    private Long userCouponId;

    private String phoneNumber;

    private String couponTitle;

    private Boolean isUsed;

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime expiredAt;

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime usedAt;

    @JsonFormat(pattern = "yyyy-MM-dd HH:mm:ss")
    private LocalDateTime receivedAt;

    private String orderNo;
}

File: SmartWash-Android/app/src/main/java/com/smartwash/network/vo/coupon/UserCouponVo.kt
package com.smartwash.network.vo.coupon

import androidx.annotation.Keep

@Keep
data class UserCouponVo(
    val userCouponId: Long,
    val phoneNumber: String,
    val couponVo: CouponVo,
    val isUsed: Boolean,
    val expiredAt: String,
    val usedAt: String,
    val receivedAt: String,
    val orderNo: String,
)